web-api/src/models.rs

```diff
@@ -452,6 +452,23 @@ pub(crate) struct PersonalizedDocument {
 
     /// The tags associated to the document.
     pub(crate) tags: DocumentTags,
+
+    /// Additional data about the document that can be helpful while tuning or debugging the system.
+    pub(crate) dev: Option<DocumentDevData>,
```

Comment:
I think we can remove this indirection.

If we ever decide to make it a stable non dev option this would add additional friction.

On the other hand a "omitted if not set" `raw_scores` field should always be fine.

Reply:
through then if we make it stable we probably add it to the document representation directly :thinking: 

so lets keep it this way

Reply:
through then even for dev options having this on the returned documents would be nice :thinking: